mem/get_malloc_size_src.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# script/process to update code from servo project (malloc_size_of)
```

Comment:
It is somehow what I did, I just keep the delta to be able to synch simply, but in the long term it may prove useless and importing code somehow will be better.